Userland/Applications/Browser/DownloadWidget.cpp

```diff
@@ -78,6 +78,8 @@ DownloadWidget::DownloadWidget(const URL& url)
 
     m_progressbar = add<GUI::Progressbar>();
     m_progressbar->set_fixed_height(20);
+    m_progressbar->set_max(0);
```

Comment:
Postmortem, but should this have been `set_min`?

Reply:
Oops, well spotted :^) I'll create a PR for it.